### PR TITLE
DOCS-3301 add documentation to track background events

### PR DESCRIPTION
**Pull request by priyanshi-gupta**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

File: docs/rum_getting_started.md [hunk 1]
```diff
@@ -326,7 +326,25 @@ This records each request processed by the `OkHttpClient` as a resource in RUM,
...
-You can also add an `EventListener` for the `OkHttpClient` to [automatically track resource timing][11] for third-party providers and network requests. 
+You can also add an `EventListener` for the `OkHttpClient` to [automatically track resource timing][11] for third-party providers and network requests.
+
+### Track background events
+
+To track events (crashes, network requests etc.) when your application is in the background, i.e. there is no active view, add the following snippet during initialization while setting up Datadog configuration:
+
+ {{< tabs >}}
+ {{% tab "Java" %}}
+ ```swift
```
Comment (0xnm) [inline docs/rum_getting_started.md:337]:
```suggestion
 ```java
```

Do we also need different tabs considering the call will look the same for both Kotlin and Java?

File: docs/rum_getting_started.md [hunk 2]
```diff
@@ -326,7 +326,25 @@ This records each request processed by the `OkHttpClient` as a resource in RUM,
...
+ .trackBackgroundRumEvents(true)
+
+ ```
+
+ <div class="alert alert-info"><p>Note that tracking background events could lead to additional sessions that might impact billing. For questions contact support@datadoghq.com</p>
+ </div>
+
+<div class="alert alert-info"><p>Note that tracking background events could lead to additional sessions that might impact billing. For questions contact support@datadoghq.com</p>
+</div>
```
Comment (0xnm) [inline docs/rum_getting_started.md:346]:
```suggestion
<div class="alert alert-info"><p>Note that tracking background events could lead to additional sessions that might impact billing. For questions contact support@datadoghq.com</p>
</div>
```

**Review by 0xnm — CHANGES_REQUESTED** [review]

File: docs/rum_getting_started.md [hunk 3]
```diff
@@ -326,7 +326,25 @@ This records each request processed by the `OkHttpClient` as a resource in RUM,
 
 **Note**: If you also use multiple Interceptors, call `DatadogInterceptor` first.
 
-You can also add an `EventListener` for the `OkHttpClient` to [automatically track resource timing][11] for third-party providers and network requests. 
+You can also add an `EventListener` for the `OkHttpClient` to [automatically track resource timing][11] for third-party providers and network requests.
+
+### Track background events
+
+To track events (crashes, network requests etc.) when your application is in the background, i.e. there is no active view, add the following snippet during initialization while setting up Datadog configuration:
```
Comment (alai97) [inline docs/rum_getting_started.md:333]:
```suggestion
You can track events such as crashes and network requests when your application is in the background (for example, no active view is available). 

Add the following snippet during initialization in your Datadog configuration:
```

File: docs/rum_getting_started.md (hunk 1 above)
Comment (alai97) [reply docs/rum_getting_started.md:337]: +1, if we are adding more tabs, please add this to the end after Line 340:

```
{{% /tab %}}
{{< /tabs >}}
```

**Review by alai97 — CHANGES_REQUESTED** [review]

**Review by 0xnm — APPROVED** [review]

**Review by xgouchet — APPROVED** [review]

**Pull request merged**